bindings/protocols/bacnet/index.html

```diff
@@ -745,10 +747,30 @@ <h3>WoT Data Schema and BACnet Types</h3>
                             <code>
                                 {
                                     "type": "string",
-                                    "pattern": "^((([1-9][0-9]*)?[0-9]{4})|\\*)-((1[0-2]|0[1-9])|\\*)-((3[01]|0[1-9]|[12][0-9])|\\*)(\\s*\\([1-7\\*]\\))?$",
-                                    "$comment": "ISO8601 Date Format, extended with day of week (1: Mon, 7: Sun) and with optional wildcards"
+                                    "pattern": "^((([1-9][0-9]*)?[0-9]{4})|\\*)-((1[0-2]|0[1-9])|O|E\\*)-((3[01]|0[1-9]|[12][0-9])|O|E\\*)(\\s*\\([1-7\\*]\\))?$",
+                                    "$comment": "ISO8601 Date Format, optionally extended with day of week (1: Mon, 7: Sun) and wildcards (O, E, *)"
                                 }
                             </code>
+
+                            <ul>
+                                <li><code>O</code> and <code>E</code> can be used in the months field to indicate odd (1: Jan, 3: Mar...) and even (2: Feb, 4: Apr...) months.</li>
+                                <li><code>O</code> and <code>E</code> can be used in the day of week field to indicate odd and even days of the month.</li>
+                                <li><code>*</code> can be used for any year, month, day of month or day of week.</li>
+                                <li>Examples
+                                    <ul>
+                                        <li><code>*-*-1</code>, first day of each month</li>
+                                        <li><code>*-6-* (1)</code>, all Mondays in June.</li>
+                                        <li><code>*-E-1</code>, first day of all even months.</li>
+                                        <li>
+                                            <code>2024-11-07 (1)</code>, an empty set, because day of week is actually Thursday (4),
+                                            the more straightforward way would be to just leave out this entry where this data would occur.
```

Comment:
```suggestion
                                            <code>2024-11-07 (1)</code>, an empty set, because the day of week for this date is actually Thursday (4).
                                            A more straightforward way would be to just leave out the entry where this data would occur.
```